ui/00_message/event/st0100/ev015.xml

```diff
@@ -1,13 +1,17 @@
 ﻿<resource>
   <message>
     <original>今度はなんだ？　奴らに聞かせるなよ</original>
+    <translation>Now what? Don't let them hear you.</translation>
   </message>
   <message>
     <original>なんだと！？
 魔物を倒しただって？</original>
+    <translation>Wait, What!?
+Did you defeat the Monster?</translation>
```

Comment:
The event text is cutscene dialogue, so a reference to a monster like this would be pretty obvious from context. I don't think it's necessary to capitalize it.